projects/igniteui-angular/src/lib/chips/chip.spec.ts

```diff
@@ -144,4 +185,167 @@ describe('IgxChip', () => {
 
         expect(secondChipComp.onRemove.emit).toHaveBeenCalled();
     });
+
+    it('should set text in chips correctly', () => {
+        const fix = TestBed.createComponent(TestChipComponent);
+        fix.detectChanges();
+
+        const chipArea = fix.debugElement.queryAll(By.directive(IgxChipsAreaComponent));
+        const chipElements = chipArea[0].queryAll(By.directive(IgxChipComponent));
+        const firstChipTextElement = chipElements[0].queryAllNodes(By.css('.igx-chip__text'));
+        const firstChipText = firstChipTextElement[0].nativeNode.innerHTML;
+
+        expect(firstChipText).toContain('Country');
+
+        const secondChipTextElement = chipElements[1].queryAllNodes(By.css('.igx-chip__text'));
+        const secondChipText = secondChipTextElement[0].nativeNode.innerHTML;
+
+        expect(secondChipText).toContain('City');
+    });
+
+    it('should set chips label correctly', () => {
+        const fix = TestBed.createComponent(TestChipsLabelAndSuffixComponent);
+        fix.detectChanges();
+
+        const chipArea = fix.debugElement.queryAll(By.directive(IgxChipsAreaComponent));
+        const chipElements = chipArea[0].queryAll(By.directive(IgxChipComponent));
+        const firstChipLabel = chipElements[0].queryAll(By.directive(IgxLabelDirective));
+        const firstChipLabelText = firstChipLabel[0].nativeElement.innerHTML;
+
+        expect(firstChipLabelText).toEqual('label');
+    });
+
+    it('should set chips prefix correctly', () => {
```

Comment:
@MartinKamenov I don't see `igxPrefix` directive in the test bed. Maybe the title of the test should be changed.